client/sites-dashboard/components/hosting-command-palette-banner.tsx

```diff
@@ -0,0 +1,238 @@
+	size: 'big' | 'small';
+}
+
+const StyledShortcut = styled.span< ShortcutIconProps >( ( props ) =>
+	props.size === 'big'
+		? {
+				fontWeight: 600,
+				fontSize: 20,
+				border: '1px solid',
+				borderRadius: 2,
+				padding: '0px 8px',
+				marginRight: 6,
+		  }
+		: {
+				fontWeight: 600,
+				fontSize: 14,
+				backgroundColor: '#EEE',
+				color: '#3C434A',
+				padding: '0 4px',
+				borderRadius: 2,
+		  }
+);
+
+const ShortcutIcon = ( { size }: ShortcutIconProps ) => {
+	const isMac = navigator.userAgent.indexOf( 'Mac' ) > -1;
+	const shortcut = isMac ? '⌘K' : 'Ctrl+K';
+	return <StyledShortcut size={ size }>{ shortcut }</StyledShortcut>;
+};
+
+const AnimatedCommand = () => {
+	const commands = useCommandsArrayWpcom( {
+		setSelectedCommandName: () => {},
+	} );
+	const prefersReducedMotion = window.matchMedia( '(prefers-reduced-motion: reduce)' ).matches;
+
+	const [ currentCommandIndex, setCurrentCommandIndex ] = useState( 0 );
+	const commandNames = useMemo( () => {
+		if ( prefersReducedMotion ) {
+			return [];
+		}
+		return [
+			'registerDomain',
+			'openReader',
+			'addNewPost',
+			'viewMySites',
+			'openHostingConfiguration',
+			'changePHPVersion',
+			'manageStagingSites',
+		].map( ( commandName ) => {
+			const command = commands.find( ( command ) => command.name === commandName );
+			return {
+				icon: command?.icon,
+				label: command?.label,
+			};
+		} );
+	}, [ commands, prefersReducedMotion ] );
+
+	useEffect( () => {
+		if ( prefersReducedMotion ) {
+			return;
+		}
+
+		const interval = setInterval( () => {
+			setCurrentCommandIndex( ( prevIndex ) => ( prevIndex + 1 ) % commandNames.length );
+		}, 3000 );
+
+		return () => clearInterval( interval );
+	}, [ prefersReducedMotion, commandNames.length ] );
+
+	if ( prefersReducedMotion ) {
+		return null;
+	}
+	return (
+		<CommandBoxWrapper>
+			{ commandNames.map( ( commandName, index ) => {
+				const prevIndex = ( currentCommandIndex - 1 + commandNames.length ) % commandNames.length;
+				if ( currentCommandIndex !== index && prevIndex !== index ) {
+					return null;
+				}
+
+				return (
+					<CommandBox
+						className={ classNames( {
+							'command-box__fadeIn': currentCommandIndex === index,
+							'command-box__fadeOut': prevIndex === index,
+						} ) }
+						key={ index }
+					>
+						{ commandName?.icon }
+						{ commandName?.label }
+					</CommandBox>
+				);
+			} ) }
+		</CommandBoxWrapper>
+	);
+};
+
+export function HostingCommandPaletteBanner( { className }: HostingCommandPaletteBannerProps ) {
+	const translate = useTranslate();
+
+	if ( hasTouch() ) {
+		return null;
+	}
+
+	return (
+		<HostingCommandPaletteBannerRoot>
+			<DismissibleCard
+				preferenceName="hosting-command-palette-banner-display"
+				className={ `hosting-command-palette-banner is-dismissible ${ className }` }
+			>
+				<ShortcutIcon size="big" />
+				<div>
+					<BannerTitle>{ translate( 'The Command Palette is here' ) }</BannerTitle>
```

Comment:
I feel like we need either `.` or `!` here. I did look at other implementations though and they don't seem to have anything at the end so perhaps that might be fine for consistency with other implementations (example: https://github.com/Automattic/wp-calypso/blob/30f0baae2fff09b2580c3a7af7c3fbc80491253f/client/my-sites/stats/stats-notices/tier-upgrade-notice.tsx#L50 )